packages/jaeger-ui-components/src/demo/trace-generators.ts

```diff
@@ -113,7 +119,8 @@ export default chance.mixin({
     });
     spans = attachReferences(spans, maxDepth, spansPerLevel);
     if (spans.length > 1) {
-      spans = setupParentSpan(spans, { startTime: timestamp, duration });
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
You can remove this line.

Reply:
If I remove the `// eslint-disable-next-line...` comment, there will be a `Do not use any type assertions` error when trying to commit as type assertions are only allowed in test files.